src/Pmi/Controller/HelpController.php

```diff
@@ -227,10 +241,12 @@ public function homeAction(Application $app)
         return $app['twig']->render('help/index.html.twig');
     }
 
-    public function videosAction(Application $app)
+    public function videosAction(Application $app, Request $request)
     {
         return $app['twig']->render('help/videos.html.twig', [
-            'videoGroups' => self::$videoGroups
+            'videoGroups' => self::$videoGroups,
+            'type' => $request->query->get('type', 'yt'),
+            'gcsBucketPath' => $app->getConfig('gcs_bucket_path')
```

Comment:
Can you add `gcs_bucket_path` to `config.yml.dist`?

Reply:
Sure, will do. Thanks!

Reply:
Also, can you change this to a more accurate name?  Maybe `help_videos_path` (since in theory, this wouldn't have to be in GCS)

Reply:
Yes, that makes sense and I also changed other variable names to be more generic.